packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -306,6 +306,7 @@ class IconButton extends StatelessWidget {
   Widget build(BuildContext context) {
     assert(debugCheckHasMaterial(context));
     final ThemeData theme = Theme.of(context);
+    final IconThemeData iconTheme = IconTheme.of(context);
```

Comment:
Replace the reference below on line 323 with this variable

Reply:
Great catch! Thanks